src/DGtal/doc/tutorials/moduleShortcuts.dox

```diff
@@ -0,0 +1,493 @@
+
+/* 
+ * Useful to avoid writing DGtal:: in front of every class.
+ */
+namespace DGtal {
+
+/**
+   
+@page moduleShortcuts  Shortcuts (for the impatient developper)
+
+@writers Jacques-Olivier Lachaud
+
+@since 1.0
+
+Part of the \ref packageTutorials.
+  
+This part of the manual describes how to use shortcuts to quickly
+create shapes and surfaces, to traverse surfaces, to save/load images
+and shapes, and to analyze their geometry.
+
+[TOC]
+
+
+The following programs are related to this documentation:
+shortcuts.cpp, shortcuts-geometry.cpp
+
+@note All rendering are made with <a href="https://www.blender.org">Blender</a>.
+@see \ref moduleIntegralInvariant for Integral Invariant estimators.
+@see \ref moduleVCM for Voronoi Covariance Measure estimators.
+
+
+@section dgtal_shortcuts_sec1 Introduction
+
+To use shortcuts, you must include the following header:
+
+\code
+#include "DGtal/helpers/Shortcuts.h"
+\endcode
+
+And choose an appropriate Khalimsky space according to the dimension
+of the object you will be processing.
+
+\code
+// Using standard 2D digital space.
+typedef Shortcuts<Z2i::KSpace> SH2;
+// Using standard 3D digital space.
+typedef Shortcuts<Z3i::KSpace> SH3;
+\endcode
+
+The general philosophy of the shorcut module is to choose reasonnable
+data structures in order to minimize the number of lines to build
+frequent digital geometry code. For instance, the following lines
+build a shape that represents the digitization of an ellipsoid.
+
+\code
+  auto params = SH3::defaultParameters();
+  // Set your own parameters with operator().
+  params( "polynomial", "3*x^2+2*y^2+z^2-90" )( "gridstep", 0.25 );
+  auto implicit_shape  = SH3::makeImplicitShape3D( params );
+  auto kspace          = SH3::makeKSpaceDigitizedImplicitShape3D( params );
```

Comment:
This test does not compile, function not def.

Reply:
It was old code. Fixed in doc.